geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -87,7 +87,9 @@ protected void applyDelta(DeltaInfo deltaInfo) {
       membersAddAll(addsDeltaInfo);
     } else {
       RemsDeltaInfo remsDeltaInfo = (RemsDeltaInfo) deltaInfo;
-      membersRemoveAll(remsDeltaInfo);
+      for (byte[] member : remsDeltaInfo.getRemoves()) {
+        memberRemove(member);
+      }
```

Comment:
Could we also update the way we're applying delta for adds so that it's consistent with how we've changed the removes, so eliminating the `memberAddAll` method and moving that logic into this method?

Reply:
Actually I decide to keep the removeAll method as it uses synchronized keyword - indicating potentially multiple threads can work on the data set. Use removeAll and addAll, no additional thread will be able to modify members concurrently -all or none.

Reply:
Good call.